commands/station.js

```diff
@@ -23,10 +23,7 @@ const panic = msg => {
 
 export const station = async ({ json, experimental }) => {
   if (!FIL_WALLET_ADDRESS) panic('FIL_WALLET_ADDRESS required')
-  if (FIL_WALLET_ADDRESS.startsWith('f1')) {
-    console.error('Warning: f1... addresses are deprecated and will not receive any rewards.')
-    console.error('Please use an address starting with f410 or 0x')
```

Comment:
Personally, I would keep the special error message to nicely explain to the current users that they have to upgrade their config.

```
panic('Error: f1... addresses are no longer supported. Please use an address starting with f410 or 0x')
```

Reply:
good idea!